application/core/database/matching-stats.go

```diff
@@ -0,0 +1,96 @@
+package database
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/koki/randommatch/entity"
+	"github.com/neo4j/neo4j-go-driver/v4/neo4j"
+	"github.com/neo4j/neo4j-go-driver/v4/neo4j/dbtype"
+)
+
+func CreateMatchingStat(MatchingStats entity.MatchingStat) (string, error) {
+	driver, err := Driver()
+	if err != nil {
+		return "", err
+	}
+	session := (*driver).NewSession(neo4j.SessionConfig{AccessMode: neo4j.AccessModeWrite})
+	defer session.Close()
+
+	uid, err := session.WriteTransaction(func(tx neo4j.Transaction) (interface{}, error) {
+		result, err := tx.Run("MERGE (n:MatchingStat {uid: $id}) "+
```

Comment:
merge is useless here, since the uid is always different so CREATE can be used directly

Reply:
CREATE generate errors

Reply:
it should not